src/app/ngrx/actions/cms.action.creator.ts

```diff
@@ -13,83 +13,11 @@ export interface CmsAccountSuccessPayload {
 }
 
 export class CmsAccountSuccessAction implements Action {
-  readonly type = ActionTypes.CMS_ACCOUNT_SUCCESS;
+  readonly type = <string>ActionTypes.CMS_ACCOUNT_SUCCESS;
   constructor(public payload: CmsAccountSuccessPayload) {}
 }
 
 export class CmsAccountFailureAction implements Action {
   readonly type = ActionTypes.CMS_ACCOUNT_FAILURE;
   constructor(public payload: any) {}
 }
-
```

Comment:
_almost_ don't have to pull from two APIs anymore, so we're getting rid of most of the CMS actions except for account, which is used to get the user image